Riot/Modules/Room/RoomViewController.m

```diff
@@ -2301,7 +2504,14 @@ - (void)sendVideoAsset:(AVAsset *)videoAsset isPhotoLibraryAsset:(BOOL)isPhotoLi
             
             // Set the chosen preset and send the video (conversion takes place in the SDK).
             [MXSDKOptions sharedInstance].videoConversionPresetName = presetName;
-            [roomInputToolbarView sendSelectedVideoAsset:videoAsset isPhotoLibraryAsset:isPhotoLibraryAsset];
+            
+            // Create or invite again the left member before sending the message in case of a discussion (direct chat)
+            [self createOrRestoreDiscussionIfNeeded:^(BOOL success) {
+                if (success)
```

Comment:
Shouldn't we display an error if this fails? Applicable for so many places, can you check them all?

Reply:
@ismailgulek we ignore here the case where the boolean `success` is false because the failure is already handled at the different requests level (for example [here](https://github.com/vector-im/element-ios/pull/6367/files#diff-4a60dc69cd6244daddcee56e737c6084a55551669d35106ee671cff7365c457eR1590), or [here](https://github.com/vector-im/element-ios/blob/develop/Riot/Modules/Application/LegacyAppDelegate.m#L2926)).
@phlniji suggests changing the completion block to handle the error code here, but this will involve too much changes in some existing methods like [createDirectChatWithUserId](https://github.com/vector-im/element-ios/blob/develop/Riot/Modules/Application/LegacyAppDelegate.m#L2911)

So I suggest keeping it like that, by adding a comment to mention the current error handling at the request level, and suggest improving the error handling in case of code rewriting. Are you ok with this suggestion?